test/test_decoding.py

```diff
@@ -10,7 +10,6 @@
 from xnmt.embedder import SimpleWordEmbedder
 import xnmt.events
 from xnmt.input_reader import PlainTextReader
-from xnmt.loss_calculator import AutoRegressiveMLELoss
```

Comment:
@msperber  once renamed MLELoss to AutoRegressiveMLELoss. Maybe he has a comment of the "renaming" this back?

Reply:
I think that's because there was a bunch of extra stuff in there that only worked for auto-regressive models. Now `MLELoss` is just a wrapper around the `calc_nll` function, and is more general.